src/harness/harness.ts

```diff
@@ -1642,7 +1647,8 @@ module Harness {
         }
 
         function writeComparison(expected: string, actual: string, relativeFilename: string, actualFilename: string, descriptionForDescribe: string) {
-            if (expected != actual) {
+            var encoded_actual = (new Buffer(actual)).toString('utf8')

```

Comment:
We already use browserify, it's one of our dev dependencies in Node, so might as well try to make it work there as well.